services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/ProtectedPathSpec.groovy

```diff
@@ -893,17 +893,22 @@ class ProtectedPathSpec extends HealthCheckSpecification {
         antiflap.portDown(protectedIsls[0].dstSwitch.dpId, protectedIsls[0].dstPort)
 
         then: "Flow state is changed to DEGRADED"
-        Wrappers.wait(WAIT_OFFSET) { assert northbound.getFlowStatus(flow.id).status == FlowState.DEGRADED }
+        Wrappers.wait(WAIT_OFFSET) {
+            flowHelper.verifyFlowState(flow.id, FlowHistoryEvent.REROUTE_FAIL_PROTECTED_PATH, FlowState.DEGRADED)
```

Comment:
This is literally false expression. You are not verifying that the reroute failed for protected path, you are just checking that a successful validation happened.